libraries/c_sdk/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -1087,6 +1087,129 @@ void _IotMqtt_ProcessSend( IotTaskPool_t pTaskPool,
 
 /*-----------------------------------------------------------*/
 
+void _IotMqtt_ManagedMqttSend( _mqttOperation_t * pOperation )
```

Comment:
I think we would need to rename this function. The name send is really confusing as this function is not responsible for the send on to network.